packages/atomic/cypress/e2e/smart-snippet-feedback-modal.cypress.ts

```diff
@@ -11,12 +11,12 @@ import {SmartSnippetSelectors} from './smart-snippet-selectors';
 describe('Smart Snippet Feedback Modal Test Suites', () => {
   function setupOpenModal() {
     new TestFixture().with(addSmartSnippet()).init();
-    SmartSnippetSelectors.feedbackDislikeButton().click();
-    SmartSnippetSelectors.feedbackExplainWhy().click();
+    SmartSnippetSelectors.feedbackDislikeButton().click({force: true});
+    SmartSnippetSelectors.feedbackExplainWhy().click({force: true});
```

Comment:
I wonder why Cypress thought these weren't visible. I'm worried that adding `force: true` would occasionally click on a non-working button.

Reply:
Yes, I am not very happy with the solution either. Think it might have something to do with the modal and its apperance with an animation. 
I did try `waitForAnimations` but it didn't fix the issue. 
To your concern about it clicking on an non-working button, it would fail further actions and subsequently the test itself. 
I have added a TODO to explore this further as needed.